chromium_src/components/sync/engine_impl/commit.cc

```diff
@@ -80,7 +103,7 @@ brave_sync::RecordsListPtr ConvertCommitsToBraveRecords(
       bookmark->site.favicon = bm_specifics.icon_url();
       bookmark->isFolder = entity.folder();
       // only mattters for direct children of permanent nodes
```

Comment:
we need more `t` letters here :)
(it seems my old typo)

Reply:
fixed